src/content.rs

```diff
@@ -14,6 +14,8 @@ use handlebars::Handlebars;
 const DOC_SEPARATOR: &str = "\n---\n";
 // Cache for page values to reduce IO on each request
 const CACHE_FILE: &str = "/config/_cache.json";
+// File that stores the timestamp at which the cache becomes invalid
+const CACHE_EXPIRY_TIME_FILE: &str = "/config/_cache_invalidator.txt";
```

Comment:
Do we need a separate file for this?

Reply:
Thanks for pointing this out. I restructured the code such that the cache file now is structured as the follows 
```
$cache_expiry_time
<JSON_of_page_data>
```

Reply:
Can it be a JSON property instead?

Reply:
It could be a property of the JSON but would be inefficient to read the entire file in cases of invalid cache. Currently it is handled as follows. 

1. Read the first line to verify cache validity. 
2. If valid, parse the rest of the file into  the page object JSON.
3. If not valid  skip reading the rest of the file.